packages/clay-card/src/Detail.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import React from 'react';
+
+import classNames from 'classnames';
+
+import Context from './Context';
+
+interface CardDetailProps extends React.HTMLAttributes<HTMLDivElement> {
+	children: React.ReactNode;
```

Comment:
Is it necessary to declare this `children` here?

Reply:
nope.